### ISSUE TITLE
sync up changes to v3.2.0 schema definition to v3.2-dev branch

### ISSUE THREAD

**Issue by karenetheridge** [OAI/OpenAPI-Specification#3400]
I was about to create a PR to copy one of my recent PRs to the v3.1.0 schema to v3.1.1 and I ran into a lot of conflicts -- it looks like a number of other recent changes still need to be synced up there.

**Comment by handrews** [comment]
Do you mean that the published 3.1.0 spec (which should not change further, aside from hopefully fixing the broken internal links Real Soon Now™) has changes which are not in the 3.1.1 spec?

**Comment by karenetheridge** [comment]
Apologies, I meant the schema definition, not the spec itself. The schema has continued to undergo revisions since 3.1.0 was released, for structural optimization as well as fixing a few errors (inconsistencies or missing bits vs the spec).

Some of the recent (as in, in the last year) changes to the 3.1.0 schema haven't been copied over to 3.1.1 or 3.2.0 yet.

It would also be nice to cut a new version of the schema (the URI contains a date) to include the fixes from the last year.

**Comment by handrews** [comment]
> It would also be nice to cut a new version of the schema (the URI contains a date) to include the fixes from the last year.

Yeah there's some confusion around the logistics - I actually already put it on the agenda for this week's TSC meeting (btw thanks for putting your PRs on the agenda last week!)

**Comment by handrews** [comment]
@karenetheridge I've changed the title to reflect the new branching structure.  Let's use this to track schema updates corresponding to ported-from-`v3.2.0-dev` PR #4188, which I think are the only "syncrhonization" needed at this point.

**Comment by lornajane** [comment]
Next step is to double check if we added to the schemas the additional security fields (a new OIDC field perhaps?) and a deprecated field - these have been added to 3.2 specification but possibly not to the schema yet?

**Comment by ralfhandl** [comment]
Missing in schema of Security Scheme Object
* oauth2MetadataUrl
* deprecated
* deviceAuthorization
* deviceAuthorizationUrl

**Comment by lornajane** [comment]
Closing as this is now fixed in #4394 (merging to a dev branch doesn't close the issue)
